### ISSUE TITLE
L-USDT transactions not shown on elements-qt UI

### ISSUE THREAD

**Issue by mpolavieja** [ElementsProject/elements#1247]
Not all transactions are swhon in elements-qt transactions (also in the csv export button).  Outgoing L-USDT transactions are not shown, except for the related tx fee and a 0.00000001 L-BTC amount.  It looks like it has to do with confidential transactions.

I am using elements 22.1.1 on windows 10 pro.

**Comment by psgreco** [comment]
Were these txs creatd in this wallet? My guess is that this is a watch-only wallet with an `importaddress`, but not enough information to unblind the transactions.
If that's the case, then the blinding key should be exported from the wallet that created the tx and then imported with `importblindingkey`

**Comment by mpolavieja** [comment]
That's right.  It sounds very much that´s the problem.  I created the wallet in specter.

I am trying to import the blinding from specter DIY device, but the problem is that is in SLIP-77 format.  Is it possible to convert it to to HEX?  Also, the importblindingkey requires two args, the blinded address and the blindingkey, that confuses me because I only have one blinding key (I don't have a blinding key for each address)

**Comment by mpolavieja** [comment]
I don't think that is the problem.   I am able to see the full details of each transaction if I use the gettransaction command in the console.  The problem is that information is not shown in the UI ("Transactions") or in the exported csv.
